notebooks/data-sources/TestGrid/metrics/ocp-ci.pipeline

```diff
@@ -642,6 +642,124 @@
               }
             }
           ]
+        },
+        {
+          "id": "973d202b-9ac0-4600-a2c8-785f01617d99",
+          "type": "execution_node",
+          "op": "execute-notebook-node",
+          "app_data": {
+            "filename": "../create_trino_tables.ipynb",
+            "runtime_image": "quay.io/aicoe/ocp-ci-analysis:latest",
+            "env_vars": [
+              "S3_ENDPOINT=http://ocs-storagecluster-cephobjectstore-openshift-storage.apps.rick.emea.operate-first.cloud/",
+              "S3_ACCESS_KEY=NZAZB79EVU4I95SWTTFQ",
```

Comment:
Can you please remove the access and secret key from this commit.

Reply:
We probably should change the keys nows. I can do that. Please reach out to me if you need the new creds